internal/service/admin/adminservice/interface.go

```diff
@@ -43,10 +82,44 @@ type adminService struct {
 	Logger          *otelzap.Logger
 }
 
-// AtomicOperation is the format waited by AtomicRepository.
+/*
+	Angus: If I recall correctly, the words for "wait" and "expect" are the same in French? They mean quite different
```

Comment:
Yes you are correct, I think some other comment I made can have this problem too, I will try to be more careful.